build/squire-raw.js

```diff
@@ -3584,15 +3584,16 @@ function Squire ( root, config ) {
     this._path = '';
     this._willUpdatePath = false;
 
-    // NATE: We are resorting to the old way of handling changes since we need to move out of noteditable nodes.
-    // To do this with the 'selectionchange' event is trickier since our current procedure triggers a selection change
-
+    // nodes.  To do this with the 'selectionchange' event is trickier since our current procedure triggers
+    // a selection change
     // if ( 'onselectionchange' in doc ) {
     //   this.addEventListener( 'selectionchange', this._updatePathOnEvent );
     // } else {
       this.addEventListener( 'keyup', this._updatePathOnEvent );
       this.addEventListener( 'mouseup', this._updatePathOnEvent );
       this.addEventListener( 'mouseup', function(){
+          console.info("MOUSEUP")
+          // debugger
```

Comment:
Strange diff here, but this appears to just be some left-over change in Editor.js that was never added to the build.